docs/source/hardware_and_software_configuration/configuring/configuring_external_webcam/configuring-external-webcam.rst

```diff
@@ -110,18 +111,38 @@ pressed to continue running the Op Mode.
 .. image:: images/DS-webcam-preview-CH-3.jpg
 
 **Important Note:** Because the Camera Stream feature is only available
-during the INIT phase of an Op Mode, you must ensure that the Vuforia
-library is activated in your Op Mode **before** the waitForStart
-command:
+during the INIT phase of an Op Mode, you must ensure that the VisionPortal
+is activated in your Op Mode **before** the waitForStart command:
 
-.. image:: images/activateBeforeWaitForStart.jpg
+.. image:: images/activateBeforeWaitForStart.png
+   :alt: The init code for the webcam must be called before waitForStart.
 
 If you do not see the Camera Stream option in your main menu on your
-DRIVER STATION, then verify that the Vuforia function is activated
+DRIVER STATION, then verify that the VisionPortal is activated
 before the waitForStart command in your Op Mode. Also make sure you’ve
 given the system enough time to initialize the Vuforia software before
 you check to see if Camera Stream is available.
 
+Scrcpy
+------
+
+To view the camera output from a computer while an OpMode is running, you
+can use `scrcpy <https://github.com/Genymobile/scrcpy>`__. To do this,
+you will first need to obtain an ADB connection with your Control Hub.
+This can be done by connecting a USB-A - USB-c cable to the USB-C port
+on your Control Hub. If on Windows, you may also connect to your Control
+Hub WiFi network and open the `REV Hardware Client <https://docs.revrobotics.com/rev-hardware-client/gs/install>`__.
+Once connected, use `these instruction <https://github.com/Genymobile/scrcpy?tab=readme-ov-file#get-the-app>`__
+to install and run scrcpy on your computer.
+
+.. image:: images/webcamWithScrcpy.jpg
+   :alt: The camera output viewed with scrcpy.
+
+**Important Note:** While scrcpy is a great way to view the camera output
+outside of competitions, the Competition Manual does now allow teams to
+have any devices other than the DRIVER STATION connected to your Control
+Hub during a match. See rule R708 for the full details.
```

Comment:
Rephrase to not point to a specific rule number. Rule numbers may shift year to year.  

Suggest something like:

> Check the Robot Construction rules in the [Competition Manual](https://ftc-resources.firstinspires.org/file/ftc/game/manual) for details.